files/provision-all

```diff
@@ -27,12 +36,17 @@ qvm-shutdown --wait sd-log && qvm-start sd-log
 all_sdw_vms_target="$(qvm-ls --tags sd-workstation --raw-list | perl -npE 's/\n/,/g' | perl -npE 's/,$//' )"
 
 # We skip dom0 in the task below, since dom0 highstate was enforced in the previous command.
+echo ".........................................................................."
 echo "Provision all SecureDrop Workstation VMs with service-specific configs"
 sudo qubesctl --show-output --max-concurrency "$max_concurrency" --skip-dom0 --targets "$all_sdw_vms_target" state.highstate
 
+echo ".........................................................................."
 echo "Add SecureDrop export device handling to sys-usb"
-# If sd-fedora-38-dvm exists it's because salt determined that sys-usb was disposable
-qvm-check --quiet sd-fedora-38-dvm 2> /dev/null && \
-  sudo qubesctl --show-output --skip-dom0 --targets sd-fedora-38-dvm state.highstate && \
+# If sd-fedora-39-dvm exists it's because salt determined that sys-usb was disposable
+qvm-check --quiet sd-fedora-39-dvm 2> /dev/null && \
+  sudo qubesctl --show-output --skip-dom0 --targets sd-fedora-39-dvm state.highstate && \
   qvm-shutdown --wait sys-usb && qvm-start sys-usb || \
   sudo qubesctl --show-output --skip-dom0 --targets sys-usb state.highstate
+
+# Shut down all VMs to ensure new configuration takes place, if the user doesn't reboot.
+qvm-ls --tags sd-workstation --raw-list | xargs qvm-shutdown --wait
```

Comment:
You might want `qvm-shutdown --force --wait`, but you might decide this is too niche a usecase and opt to leave it as is. The use case would be: if the user had a running appvm that used sd-whonix for networking (eg they were checking the admin interface), then sd-whonix wouldn't reboot without the `force` command.

Reply:
I removed this commit, https://github.com/freedomofpress/securedrop-workstation-ci/pull/44 uses force so not an issue, but if we want to tweak it further, probably should go in the CI repo.